packages/api/tests/lambdas/test-process-s3-dead-letter-archive.js

```diff
@@ -135,6 +157,23 @@ test('processDeadLetterArchive only deletes dead letters that process successful
   t.deepEqual(output.processingFailedKeys, [failingMessageKey]);
 });
 
+test.serial('processDeadLetterArchive processes a SQS Message', async (t) => {
+  const { bucket, sqsPath, SQSCumulusMessage } = t.context;
+  const writeRecordsFunctionSpy = sinon.spy();
+
+  const expected = { ...SQSCumulusMessage };
+  expected.cumulus_meta.workflow_stop_time = SQSCumulusMessage.stopDate;
+
+  await processDeadLetterArchive({
+    bucket,
+    path: sqsPath,
+    writeRecordsFunction: writeRecordsFunctionSpy,
+  });
+
+  const messageArgs = writeRecordsFunctionSpy.getCalls().map((call) => call.args[0].cumulusMessage);
+  t.deepEqual(messageArgs[0], SQSCumulusMessage);
```

Comment:
```suggestion
  t.deepEqual(writeRecordsFunctionSpy.getCall(0).firstArg.cumulusMessage, SQSCumulusMessage);
```

Is there a need for `.map` when we're only expecting one call?

Reply:
No, my sinon-fu was just not strong that day.    Thanks for the amend.